plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/util/Validator.js

```diff
@@ -33,8 +34,42 @@ Ext.define('NX.util.Validator', {
   },
 
   /**
+   * @private
+   */
+  nxNameRegex : /^[^(_)#A-Z\\\/"'\*\?<>|\s,.][^#A-Z\\\/"'\*\?<>|\s,.]*$/,
```

Comment:
Note that this still accepts names containing reserved URL characters: http://en.wikipedia.org/wiki/Percent-encoding#Percent-encoding_reserved_characters, for example: "build&test". I assume these characters will be percent-encoded when we construct URLs based on such names?